src/styles/components/card.css

```diff
@@ -0,0 +1,100 @@
+.card {
+  display: flex;
+  flex-direction: column;
+
+  max-width: 200px;
+  padding: 16px;
+  background: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  font-family: "Roboto", sans-serif;
+  font-style: normal;
+}
+
+.card__image {
+  width: 160px;
+  height: 134px;
+  margin: 16px auto 0;
+}
+
+.card__title {
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 18px;
+  color: #060b35;
```

Comment:
it will be good to make variables for colors)
and check this to make your css more readable https://codeguide.academy/html-css.html#css-order